nmap.go

```diff
@@ -19,12 +20,16 @@ type ScanRunner interface {
 
 // Scanner represents an Nmap scanner.
 type Scanner struct {
+	Cmd *exec.Cmd
```

Comment:
The `exec.Cmd`, stderr and stdout should not be exposed, I'd suggest adding methods to get them safely from the `Scanner` object :)

Reply:
Will do.